infoschema/tables.go

```diff
@@ -1486,6 +1489,17 @@ var tablePlacementPoliciesCols = []columnInfo{
 	{name: "LEARNERS", tp: mysql.TypeLonglong, size: 64},
 }
 
+var tableVariablesInfoCols = []columnInfo{
+	{name: "VARIABLE_NAME", tp: mysql.TypeVarchar, size: 64, flag: mysql.NotNullFlag},
+	{name: "VARIABLES_SCOPE", tp: mysql.TypeVarchar, size: 64, flag: mysql.NotNullFlag},
```

Comment:
`VARIABLES_SCOPE` -> `VARIABLE_SCOPE`

Is it a typo?

Reply:
Yeah, I agree it should be variable scope.